scripts/src/main/bash/stats-redirects-test.sh

```
@@ -0,0 +1,59 @@
+FAILED_TESTS=()
+
+echo "=========================================="
+echo "Fetching languages with @mappings..."
+
+MAPPINGS_PAGE=$(curl -s "${BASE_URL}/mappings/")
+[ -z "$MAPPINGS_PAGE" ] && { echo "❌ ERROR: Could not fetch mappings page"; exit 1; }
+
+LANGUAGES=($(echo "$MAPPINGS_PAGE" | grep -oP 'href="\K[^/"]+(?=/">)' | sort -u))
+[ ${#LANGUAGES[@]} -eq 0 ] && { echo "❌ ERROR: No languages found"; exit 1; }
+
+echo "Found ${#LANGUAGES[@]} languages: ${LANGUAGES[@]}"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)